docs/standard/garbage-collection/fundamentals.md

```diff
@@ -1,42 +1,42 @@
 ---
 title: "Fundamentals of garbage collection"
 description: "Learn how the garbage collector works and how it can be configured for optimum performance."
-ms.date: "03/08/2018"
+ms.date: 11/11/2019
 ms.technology: dotnet-standard
 helpviewer_keywords:
   - "garbage collection, generations"
-  - "garbage collection, background garbage collection"
-  - "garbage collection, concurrent garbage collection"
-  - "garbage collection, server garbage collection"
-  - "garbage collection, workstation garbage collection"
+  - "garbage collection, background"
+  - "garbage collection, concurrent"
+  - "garbage collection, server"
+  - "garbage collection, workstation"
   - "garbage collection, managed heap"
 ms.assetid: 67c5a20d-1be1-4ea7-8a9a-92b0b08658d2
 ---
 # Fundamentals of garbage collection
 
-In the common language runtime (CLR), the garbage collector serves as an automatic memory manager. It provides the following benefits:
+In the common language runtime (CLR), the garbage collector (GC) serves as an automatic memory manager. It provides the following benefits:
```

Comment:
One question I had while reading this article (Hopefully @Maoni0 can answer):  Should I assume that mobile apps built with Xamarin use the Workstation GC? We should state that explicitly.

Reply:
See line 172.